FWCore/Framework/src/Schedule.cc

```diff
@@ -605,6 +605,7 @@ namespace edm {
     std::map<std::string, std::vector<std::pair<std::string, int>>> outputModulePathPositions;
     reduceParameterSet(proc_pset, tns.getEndPaths(), modulesInConfig, usedModuleLabels, outputModulePathPositions);
     {
+      //CDJ: needs SignallingProductRegistryFiller
```

Comment:
Just to double check, is this comment relevant to kept (as opposed to your note during development)?

Reply:
removed